### ISSUE TITLE
about blind image SR

### ISSUE THREAD

**Issue by CodeMadUser** [hellloxiaotian/ESRGCNN#8]
hello PhD. TIAN,it's an amazing work.But when I tested the blind image super-resolution, I didn't get the same results as the low-resolution image.My folder directory is as follows:
![image](https://user-images.githubusercontent.com/97572749/199481014-3bfd67a8-9d1a-46cb-874f-586e2f28f55e.png)

I used this directive：python tcw_sample_b.py --model esrgcnn --test_data_dir dataset/Set5 --scale 2 --ckpt_path checkpoint/esrgcnn.pth --sample_dir samples_singlemodel_urban100_x2
the result is:
![image](https://user-images.githubusercontent.com/97572749/199480908-271c588c-0d41-40a2-a84f-696b604c1d65.png)
I didn't get the results for img_006_SRF_2_LR.png.I don't know why,I'm looking forward to your replying.thank you.
How do I test blind graphics for super resolution?

**Comment by hellloxiaotian** [comment]
Thank you for your concern.
There is only a single image called img_006_SRF_2_LR.png, but the corresponding image img_006_SRF_2_HR.png is missing. So the dataset can't load the img_006_SRF_2_LR.png.

**Comment by CodeMadUser** [comment]
> 

Thank you for your reply. And that is that this code is temporarily unavailable the real world blind super resolution , right?

**Comment by hellloxiaotian** [comment]
Yes, you can modify the test_dataset class to make it available.
